### PR TITLE
65538 - Hide financial account when he's inactive on grid selection

### PR DESCRIPTION
**Pull request by paa-axelor**

### CODE REVIEW

File: axelor-base/src/main/resources/views/AccountManagement.xml [hunk 1]
```diff
@@ -32,7 +32,7 @@
       form-view="fixed-asset-category-form" grid-view="fixed-asset-category-grid" canEdit="false"/>
     <field name="purchaseAccount" if-module="axelor-account" form-view="account-form"
       grid-view="account-grid" if="__config__.app.isApp('account')" canEdit="false"
-      domain="self.company = :company AND (self.accountType.technicalTypeSelect = 'charge' OR self.accountType.technicalTypeSelect = 'debt' OR self.accountType.technicalTypeSelect = 'immobilisation')"/>
+      domain="self.company = :company AND (self.accountType.technicalTypeSelect = 'charge' OR self.accountType.technicalTypeSelect = 'debt' OR self.accountType.technicalTypeSelect = 'immobilisation') AND self.statusSelect = 1"/>
```
Comment (aol-axelor) [inline axelor-base/src/main/resources/views/AccountManagement.xml:35]: Can you factorize the part about `self.accountType.technicalTypeSelect` with an array please

File: changelogs/unreleased/65538.yml [hunk 2]
```diff
@@ -0,0 +1,3 @@
+---
+title: "Product/Account Management : Hide financial account when he's inactive on product account management"
```
Comment (aol-axelor) [inline changelogs/unreleased/65538.yml:2]: Please use `it` instead of `he`

File: axelor-base/src/main/resources/views/AccountManagement.xml (hunk 1 above)
Comment (paa-axelor) [reply axelor-base/src/main/resources/views/AccountManagement.xml]: Resolved

File: changelogs/unreleased/65538.yml (hunk 2 above)
Comment (paa-axelor) [reply changelogs/unreleased/65538.yml]: Resolved

**Pull request merged**